pynest/examples/Potjans_2014/network.py

```diff
@@ -0,0 +1,506 @@
+        self.weight_mat = get_weight(
+            self.net_dict['PSP_mean_matrix'], self.net_dict
+            )
+        self.weight_mat_std = self.net_dict['PSP_std_matrix']
+        self.w_ext = self.w_from_PSP
+        if self.net_dict['poisson_input']:
+            self.DC_amp_e = np.zeros(len(self.net_dict['populations']))
+        else:
+            if nest.Rank() == 0:
+                print(
+                    '''
+                    no poisson input provided
+                    calculating dc input to compensate
+                    '''
+                    )
+            self.DC_amp_e = compute_DC(self.net_dict, self.w_ext)
+
+        if nest.Rank() == 0:
+            print(
+                'The number of neurons is scaled by a factor of: %.2f'
+                % self.N_scaling
+                )
+            print(
+                'The number of synapses is scaled by a factor of: %.2f'
+                % self.K_scaling
+                )
+
+        # Scaling of the synapses.
+        if self.K_scaling != 1:
+            synapses_indegree = self.synapses / (
+                self.N_full.reshape(len(self.N_full), 1) * self.N_scaling)
+            self.weight_mat, self.w_ext, self.DC_amp_e = adj_w_ext_to_K(
+                synapses_indegree, self.K_scaling, self.weight_mat,
+                self.w_from_PSP, self.DC_amp_e, self.net_dict, self.stim_dict
+                )
+
+        # Create cortical populations.
+        self.pops = []
+        pop_file = open(
+            os.path.join(self.data_path, 'population_GIDs.dat'), 'w+'
+            )
+        for i, pop in enumerate(self.net_dict['populations']):
+            population = nest.Create(
+                self.net_dict['neuron_model'], int(self.nr_neurons[i])
+                )
+            nest.SetStatus(
+                population, {
+                    'tau_syn_ex': self.net_dict['neuron_params']['tau_syn_ex'],
+                    'tau_syn_in': self.net_dict['neuron_params']['tau_syn_in'],
+                    'E_L': self.net_dict['neuron_params']['E_L'],
+                    'V_th': self.net_dict['neuron_params']['V_th'],
+                    'V_reset':  self.net_dict['neuron_params']['V_reset'],
+                    't_ref': self.net_dict['neuron_params']['t_ref'],
+                    'I_e': self.DC_amp_e[i]
+                    }
+                )
+            self.pops.append(population)
+            pop_file.write('%d  %d \n' % (population[0], population[-1]))
+        pop_file.close()
+        for thread in np.arange(nest.GetKernelStatus('local_num_threads')):
+            local_nodes = nest.GetNodes(
+                [0], {
+                    'model': self.net_dict['neuron_model'],
+                    'thread': thread
+                     }, local_only=True
+                )[0]
+            vp = nest.GetStatus(local_nodes)[0]['vp']
+            # same thread on MPI proc -> same vp
```

Comment:
I adopted this comment and the code to initialize the membrane potential from jhnnsnk. This comment means - if I understand it correctly (and the code above) - that the nest.GetStatus command is called for all local nodes in one thread. Then the vp of the first node is extracted to initialize the membrane potential. This is safe, because only local nodes on the same thread are retrieved. Then this is done for the next thread. 
So the comment should maybe elaborate this a little bit more: 
--> vp the same for all local nodes on the same thread